pipelines/deployer/deployer.tf

```diff
@@ -57,6 +57,11 @@ variable "minimum_workers_per_az_count" {
   type = string
 }
 
+variable "desired_workers_per_az_map" {
+  type    = string
```

Comment:
Since we are using a tfvars _file_ to pass in this variable ... maybe we can use a `map` type here instead of a string type and avoid the stringly typedness.

https://www.terraform.io/docs/configuration/variables.html#complex-typed-values